client/dashboard/data/domain-nameservers.ts

```diff
@@ -0,0 +1,27 @@
+import wpcom from 'calypso/lib/wp';
+
+export async function fetchDomainNameservers( domainName: string ): Promise< string[] > {
+	return wpcom.req.get( {
+		path: `/domains/${ domainName }/nameservers/`,
+	} );
+}
+
+export async function updateDomainNameservers(
+	domainName: string,
+	nameservers: string[]
+): Promise< string[] > {
+	return wpcom.req
+		.post( {
+			path: `/domains/${ domainName }/nameservers`,
+			body: {
+				nameservers: nameservers.map( ( nameserver ) => ( { nameserver } ) ),
+			},
+		} )
+		.then( ( response: { success: boolean } ) => {
```

Comment:
Are we not including the updated entity in the response? Weird.

Ideally we return that instead of the input.

Reply:
Currently, the response looks like:
```
{ success: boolean }
```